File: Week2/Graders/grader.js
//Simple grader by jake coffey 4/17/22 
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
  });
function check(grade){
    let valid=true;
    if(grade>100||grade<0){
        console.log("Grade must be between 0 and 100!");
        valid = false;
    }
    
    return valid;
}

readline.question('Please input student name:', (student) => {

        readline.question(`Please enter ${student}s grade:`, (stGrade) => {
            if(check(stGrade)){
                if(stGrade<60){
                    console.log(`${student} got an D :(`); 
                }
                else if (stGrade<70&&stGrade>=60){
                    console.log(`${student} got a D :/`); 
                }
                else if (stGrade<80&&stGrade>=70){
                    console.log(`${student} got a C :L`); 
                }
                else if (stGrade<90&&stGrade>=80){
                    console.log(`${student} got a B :)`); 
                }
                else  {
                    console.log(`${student} got an A :)!!`); 
                }
            }
        readline.close();
        });
    });

File: Week2/Graders/grader.py
#Must be run in python 3!
#Simple grader by jake coffey 5/15/22 

student= input("please enter the students name:")
loop = True
while(loop):
    strGrade=input(f' please input {student}s grade:')
    intGrade= float(strGrade)
    if intGrade>100 or intGrade<0 :
        print ("Grades must be between 0 and 100!")
    
    else:
        loop=False
    

if intGrade<60:
    print (f'{student} got a F :(')

elif intGrade>=60 and intGrade<70 :
    print( f'{student} got a D :/')

elif intGrade>=70 and intGrade <80 :
    print (f'{student} got a C :L')

elif intGrade>=80 and intGrade<90 :
    print (f'{student} got a B :)')

elif intGrade>=90 and intGrade<100 :
    print (f'{student} got an A  :)!')